spacemesh/v2/node.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package spacemesh.v2;
+
+message NodeStatusRequest {}
+
+message NodeStatusResponse {
+  uint64 connected_peers = 1; // number of connected neighbors
+  bool   is_synced = 2; // true when meshed is synced
+  uint32 synced_layer = 3; // the last layer node has synced
+  uint32 verified_layer = 4; // the last layer node has verified
+  uint32 head_layer = 5; // head layer is the tip
```

Comment:
how is this different than verified_layer?

Reply:
`verified_layer` is the latest verified layer by the node
`head_layer` will be the top layer, based on time